scripts/core-triage/project.py

```diff
@@ -0,0 +1,198 @@
+# imports
+import os
+import requests
+
+# TODO: improve
+from graphql_queries import *
+
+# TODO: what is error handling?
+
+# constants # TODO: move to a config file (or something) <= copilot suggested this last part
+ORG = "dbt-labs"
+PROJECT_NAME = "Core triage"
+PROJECT_NUM = 22
+# core team name
+CORE_TEAM = "core"
+# repos determined by querying the team
+# exclude some repos (generally internal) that will cause failure
+# include some extra repos not in the team
+EXCLUDE_REPOS = ["core-team", "schemas.getdbt.com"]
+EXTRA_REPOS = ["dbt-starter-project", "jaffle_shop"]
```

Comment:
Nothing to change - just a question.  Why are we tracking repos that we don't own?  Maybe a better question is why do we not own these if we want to track them?

Reply:
good question -- going to resolve this so I can merge, but those two are both essential (I think) for the getting started experience with dbt, one used in `dbt init` and the other as the canonical example for learning. I think that getting started experience is fundamental to Core. I think as we start to think about supporting data science workflows, it becomes even more important (standing up new projects based on, perhaps cookiecutter, templates).

there's no harm in adding these to the project and we can filter them out (I think they are now) in the project views, but I'd like to have them for personal curiosity to check if users are opening issues on them

Reply:
This was def not a blocker, more curiosity of your thinking here.  Also agree there's no harm.

Note: `dbt-starter-project` is used *just* by cloud now.  For anyone else using dbt `dbt-init` actually uses the internal [starter project](https://github.com/dbt-labs/dbt-core/tree/main/core/dbt/include/starter_project).